.github/actions/release/changelog-generator/action.yml

```diff
@@ -43,13 +59,28 @@ runs:
           --project ${{ env.GITHUB_REPOSITORY_PROJECT }} 
           --token ${{ inputs.github-token }} 
           --future-release ${{ inputs.future-release }} 
+          ${{ env.SINCE_ARG }}
           ${{ inputs.optional-arg }}
 
     - name: Populate changelog descriptions
       shell: bash
       run: |
         .github/workflows/scripts/populate-changelog-descriptions.sh ${{ inputs.changelog-path }}
 
+    - name: Cut other releases
+      if: ${{ inputs.output-only-current-release }}
+      shell: bash
+      run: |
+        ESCAPED_CURRENT_VERSION=${${{ inputs.previous-release }}//./\\.}
+        # cut everything before the current release changes
+        sed -n -z -E "s/.*$ESCAPED_CURRENT_VERSION\)(.*)/\1/p" -i ${{ inputs.changelog-config-path }}
+        
+        if [[ ${{ inputs.previous-release }} ]]; then
+          ESCAPED_PREVIOUS_VERSION=${${{ inputs.previous-release }}//./\\.}
+          # cut everything after the current release changes based on the previous version entry
```

Comment:
Both `ESCAPED_CURRENT_VERSION` and `ESCAPED_PREVIOUS_VERSION` use `inputs.previous-release`. Is that meant to be `inputs.future-release` instead for `ESCAPED_CURRENT_VERSION`?

Reply:
good catch